sdk/storage/Azure.Storage.Queues/src/QueueClientOptions.cs

```diff
@@ -30,7 +30,12 @@ public enum ServiceVersion
             /// The 2019-02-02 service version described at
             /// <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/versioning-for-the-azure-storage-services#version-2019-02-02" />
             /// </summary>
-            V2019_02_02 = 1
+            V2019_02_02 = 1,
+
+            /// <summary>
+            /// The 2019-07-07 service version.
+            /// </summary>
+            V2019_07_07 = 2,
```

Comment:
Is there a corresponding service version documented online?  We shouldn't add a version that doesn't do anything.

Reply:
(Should fix)

Reply:
This is the service version for STG71.  There is no documentation because it will be released next week.

Reply:
I get that for Blobs/Files, but will there be documentation for Queues next week?  I'm asking because we didn't add any APIs to Queues and I'm wondering what this enum value would mean to customers.

Reply:
Nothing was added in Queues, but the service version is also being bumped for Queues.  I just asked Peter about this.